pkgs/build-support/setup-hooks/autoreconf.sh

```diff
@@ -1,7 +1,7 @@
 preConfigurePhases+=" autoreconfPhase"
 
 for i in @autoconf@ @automake@ @libtool@ @gettext@; do
-    findInputs $i nativePkgs propagated-native-build-inputs
+    findInputs $i -1
```

Comment:
Could you describe the use of -1 in a comment? It's not clear when reading this.